src/Authoring/TestDiagnostics/Scenarios.cs

```diff
@@ -0,0 +1,60 @@
+﻿using ABI.System.Numerics;
+using ABI.Windows.ApplicationModel.Contacts;
+using System;
+using System.Runtime.ExceptionServices;
+using Windows.Foundation;
+using Windows.Foundation.Numerics;
+using Windows.Security.Cryptography.Core;
+using Windows.Web.Syndication;
+
+namespace TestDiagnostics
+{
+    /*

```

Comment:
I think I see why you have all the invalid scenarios commented out (so that they don't trigger and break build).  But I think we still want to be able to run tests to validate none of the checks regress so we should consider creating a compilation context in a text for a certain source text and looking for diagnostics.

Reply:
yep coming now